src/context/CartContext.js

```diff
@@ -0,0 +1,27 @@
+import { createContext,useContext,useReducer } from "react";
+
+const CartContext=createContext();
+
+const CartProvider=({children})=>{
+
+    const cartReducerFunc=(cartState,action)=>{
```

Comment:
reducer can be extracted out and can be imported wherever required

Reply:
sure will do this.